fs/react-next-base
# Refactoring. Update jest for compatibility with ts-jest

# Summary

#### A brief description of the pull request.

#### Resolves: #issue_number

#### Screenshots in case of UI changes

# Test plan

#### List of steps to manually test introduced functionality:

- Checkout to branch locally
- Run test, you shouldn't see error: `ts-jest[versions] (WARN) Version 25.5.4 of jest installed has not been tested with ts-jest. If you're experiencing issues, consider using a supported version (>=26.0.0 <27.0.0-0). Please do not report issues in ts-jest if you are using unsupported versions.`


#### What do need to pay more attention to, some implicit changes that influence the app:

- ...

# Review notes

#### While reviewing pull-request (especially when it's your pull-request), please make sure that:

- you named PR close to issue name and linked that issue
- you understand what problem is solved by PR and how is it solved
- new tests are in place, no redundant tests
- required ENV variables added and described in `.env.example` and added to Heroku
- associated Heroku review app works correctly with introduced changes
